src/lib/components/Modal.svelte

```diff
@@ -46,7 +46,7 @@
 		role="presentation"
 		tabindex="-1"
 		bind:this={backdropEl}
-		on:click={handleBackdropClick}
+		on:click|stopPropagation={handleBackdropClick}
```

Comment:
just to be sure, this doesn't affect other use cases of `Modal`. Right ?

Reply:
I tested our other modals and it doesn't, but I think we needed this line anyway: otherwise when you have two modals, clicking the backdrop would close all of them instead of the topmost one